README.md

```diff
@@ -139,10 +139,22 @@ If the above requirements are not met (external devices cannot resolve Kubernete
 
 ### Deploy Clearwater in Kubernetes
 
+#### Using kubectl
+
 To deploy the images, you should simply run `kubectl apply -f clearwater-docker/kubernetes`.  It may take a minute or so before the deployment is fully established, the load balancer is created, and the deployment is ready to accept calls.
 
 Note this will deploy all containers.  If you don't need e.g. Bono, Homestead-prov, Ellis etc. then just move the corresponding svc and depl files out of the directory before running the create command.
 
+#### Using helm
+
+To create a helm chart, you should simply run `./k8s-genchart`.
+
+If you want to use your own docker registry, modify `clearwater/values.yaml`.
```

Comment:
If you're able to make the changes to build Helm config using the k8s-gencfg script, we won't need this :)
I'm assuming users will still have had to the up the config-map, as in the steps above? Haven't got helm installed on the rig i have available so haven't been able to test myself

Reply:
We may include the ConfigMap with default values in helm chart. It is quite convenient to override the default values from command arguments.